packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -1153,6 +1153,7 @@ The following conditions may be applied to the global session policy.
 | ---                     | ---                                                                                                                                                                       | ---                                             | ---                           | ---     |
 | access                  | `ALLOW` or `DENY`                                                                                                                                                         | `ALLOW` or `DENY`                               | Yes                           | N/A     |
 | requireFactor           | Indicates if multifactor authentication is required                                                                                                                      | Boolean                                         | No                            | false   |
+| primaryFactor <ApiLifecycle access="ie" /> | Indicates the primary factor used to establish a session for the org. Possible values: `PASSWORD_IDP_ANY_FACTOR` (any factor can be used to meet the authentication policy), `PASSWORD_IDP` (only a password can be used to meet the authentication policy)  | String | Yes, if `access` is set to `ALLOW`                        | N/A   |
```

Comment:
Instead of `any factor can be used to meet the authentication policy` maybe should say `any factor used to meet the authentication policy of the app` - In OIE, you are always logging into some app (we refer to app sign on policies are as authentication policies). PASSWORD_IDP_ANY_FACTOR means anything the app's authentication policy requires can be used to establish a session.

I would then change `only a password can be used to meet the authentication policy` to `a password must always be used`.